//
// Copyright 2019 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//

syntax = "proto2";

package zetasql;

option java_package = "com.google.zetasql";
option java_outer_classname = "ZetaSQLParser";

message ParseResumeLocationProto {
  // The filename associated with the input string (if any).
  optional string filename = 4;
  // The input string.
  optional string input = 1;
  // The current byte position, the parser will resume from this position in the
  // next round.
  optional int32 byte_position = 2;
  optional bool allow_resume = 3;
}
